src/net/sourceforge/kolmafia/preferences/Preferences.java

```diff
@@ -55,6 +55,8 @@ public class Preferences {
 
   private static final Set<String> defaultsSet = new HashSet<>();
   private static final Set<String> perUserGlobalSet = new HashSet<>();
+  private static final Set<String> resetOnRollover =
```

Comment:
I think this would be better named doesntResetOnAscension or onlyResetOnRollover since everything with a leading _ sounds like it should be here